cmd/skipper/main.go

```diff
@@ -172,6 +174,7 @@ func init() {
 	flag.StringVar(&applicationLogPrefix, "application-log-prefix", defaultApplicationLogPrefix, applicationLogPrefixUsage)
 	flag.StringVar(&accessLog, "access-log", "", accessLogUsage)
 	flag.BoolVar(&accessLogDisabled, "access-log-disabled", false, accessLogDisabledUsage)
+	flag.BoolVar(&accessLogJSONDisabled, "access-log-json-disabled", true, accessLogJSONDisabledUsage)
```

Comment:
i would prefer to use accessLogJSONEnabled or just simply accessLogJSON.

the reason is that this way the default behavior for the binary distribution, built from cmd/skipper, stays the same, but not so for those projects that import skipper as a library, from the root of the project. This way it would not be backwards compatible for the lib, only for the cmd.